lib/v2/screens/explore_screens/invite/components/invite_link_dialog.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:qr_flutter/qr_flutter.dart';
+import 'package:seeds/v2/components/custom_dialog.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+import 'package:seeds/v2/domain-shared/ui_constants.dart';
+import 'package:seeds/i18n/plant_seeds.i18n.dart';
+import 'package:seeds/v2/screens/explore_screens/invite/interactor/viewmodels/bloc.dart';
+
+class InviteLinkDialog extends StatelessWidget {
+  const InviteLinkDialog({Key? key}) : super(key: key);
```

Comment:
do we need this ?

Reply:
constructor. 
The comment is left in line 13. 
its empty constructor

Reply:
Well the `const` constructor is necessary for the performance from there that it is refactorized in widgets.
So that when you create an instance you create it with `const` as it happens with the `const SizedBox()` 
The framework reuses these constant widgets instead of creating them.

What I think it is not necessary is the key parameter, this I only put it because of habit also because it is what my generator of templates puts.

Reply:
cool!

Reply:
im good with the PR> 
Is up to @gguij004  now